src/app/src/main/kotlin/org/vaccineimpact/api/app/controllers/OneTimeLinkController.kt

```diff
@@ -124,7 +126,8 @@ class OneTimeLinkController(
 
         val claims = try
         {
-            tokenHelper.verify(token, TokenType.LEGACY_ONETIME)
+            tokenHelper.verify(token, TokenType.LEGACY_ONETIME,
+                    NoopOneTimeTokenChecker()) // token has already been checked in previous step
```

Comment:
This is not that nice, but it's just to keep supporting the old style onetime tokens. It will go away once those are deprecated.